packages/salesforcedx-vscode-core/test/vscode-integration/util/inputUtils.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { createSandbox, SinonSandbox } from 'sinon';
+import * as vscode from 'vscode';
+import { getTrimmedString } from '../../../src/util/inputUtils';
+
+describe('getTrimmedString', () => {
+  let sandbox: SinonSandbox;
+  const INPUT_VAL = 'Test Input';
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should call showInputBox once', async () => {
+    const inputBox = sandbox.stub(vscode.window, 'showInputBox').resolves(INPUT_VAL);
```

Comment:
you should follow the same pattern here that you do in the other test.  Create the stub in the before each.

Reply:
oh also the same naming nit.  I usually use the thing I'm stubbing with a postfix of Stub: `showInputBoxStub`.